ogre2/include/gz/rendering/ogre2/Ogre2GzOgreRenderingMode.hh

```diff
@@ -29,27 +29,32 @@ namespace gz
       /// \brief Rendering modes so that GzHlms implementations
       /// follow alternate code paths or extra customizations
       /// when they're enabled
-      enum IgnOgreRenderingMode
+      enum GzOgreRenderingMode
       {
         /// \brief Normal rendering
-        IORM_NORMAL,
+        GORM_NORMAL = 0,
+        IORM_NORMAL = 0,  // TODO(CH3): Deprecated. Remove on tock.
 
         /// \brief Render a solid color explicit per object.
         /// Used by e.g. Segmentation camera mode
-        IORM_SOLID_COLOR,
+        GORM_SOLID_COLOR = 1,
+        IORM_SOLID_COLOR = 1,  // TODO(CH3): Deprecated. Remove on tock.
 
-        /// \brief Like IORM_SOLID_COLOR, but if CustomParameter 2u
+        /// \brief Like GORM_SOLID_COLOR, but if CustomParameter 2u
         /// is present, raw diffuse texture will be multiplied against
         /// the solid colour.
         ///
-        /// Also Unlit will behave as if IORM_NORMAL
+        /// Also Unlit will behave as if GORM_NORMAL
         ///
         /// Used by thermal camera
-        IORM_SOLID_THERMAL_COLOR_TEXTURED,
+        GORM_SOLID_THERMAL_COLOR_TEXTURED = 2,
+        IORM_SOLID_THERMAL_COLOR_TEXTURED = 2,  // TODO(CH3): Deprecated. Remove on tock.
 
         /// \brief Total number of rendering modes
-        IORM_COUNT,
+        GORM_COUNT = 3,
+        IORM_COUNT = 3,  // TODO(CH3): Deprecated. Remove on tock.
       };
+      using IgnOgreRenderingMode GZ_DEPRECATED(7) = GzOgreRenderingMode;  // TODO(CH3): Deprecated. Remove on tock.
```

Comment:
Windows doesn't like this. How about adding ifdefs so this isn't defined on Windows? Windows isn't an officially supported platform, so I think it's ok we don't provide some tick-tocks there.

Reply:
https://github.com/gazebosim/gz-rendering/pull/672/commits/91f00c749ebfdcbd5e54306d1a0af178e364b7ea I think I did it right?